src/auxiliary/mpi_arrays.jl

```diff
@@ -216,13 +221,15 @@ Instead of checking something like `any(isnan, u)`, this function just checks
 global communication is required and all ranks will return the same result.
 
 You should pass this function as keyword argument
-`unstable_check=Trixi.ode_unstable_check`
+`unstable_check=ode_unstable_check`
 of `solve` when using error-based step size control with MPI parallel execution
 of Trixi.jl.
+
+See [`https://diffeq.sciml.ai/stable/basics/common_solver_opts/#Miscellaneous`](https://diffeq.sciml.ai/stable/basics/common_solver_opts/#Miscellaneous)
 """
 ode_unstable_check(dt, u, semi, t) = isnan(dt)
 
 
 end # module
 
-using .TrixiMPIArrays: TrixiMPIArrays, TrixiMPIArray, ode_norm, ode_unstable_check
+@reexport using .TrixiMPIArrays: TrixiMPIArrays, TrixiMPIArray, ode_norm, ode_unstable_check
```

Comment:
`ode_norm` and `ode_unstable_check` are rather generic terms. This is no issue as long as they are used only internally, but I am wondering if we should rather make them more specific, e.g., `ode_norm_parallel` or similar (to avoid clashes with other packages and to make it clear when to use them).

OTOH, we can also just keep it as it is for now and revisit once we de-experimentalize the TrixiMPIArray implementation.

Reply:
Yeah, that's also what I would like to do. `ode_norm` also works in serial, so I don't like `ode_norm_parallel`. I marked everything as experimental, so we can change it later.